pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/themes/public/themes/sapphire/globalSapphire.css

```diff
@@ -6254,7 +6254,7 @@ div.dijitDateTextBox.dijitTextBox {
 #DP_range2_lookup,
 div.dijitDateTextBox input.dijitInputInner[type='text'] {
   padding-right: 0px;
-  padding-top: 0px;
+  padding-top: 0px !important;

```

Comment:
This is not a good fix. Why do we need !important here if we don't need it in Crystal?